src/main/java/com/gradle/CustomBuildScanEnhancements.java

```diff
@@ -61,30 +61,45 @@ private void captureIde() {
         if (!isCi()) {
             // Wait for projects to load to ensure Gradle project properties are initialized
             gradle.projectsEvaluated(g -> {
+                Optional<String> ideaVendorName = sysProperty("idea.vendor.name");
```

Comment:
Cool that there seems to be a relevant property now ! Do you know if it's a recent addition ? I asked a similar question a year ago to Jetbrains and they could not give me such property: https://youtrack.jetbrains.com/issue/IDEA-279187#focus=Comments-27-5233519.0-0

Reply:
This is the information I got

> The idea.vendor.name is available approximately since 2020.1 release

Reply:
Nice 👍 !